README.md

```diff
@@ -26,14 +26,26 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+React is a UI library that is used in various forms to create complex, rich user interfaces. It makes it easier to write javascript and pass multiple components into file.
+
 - [ ] What does it mean to _think_ in react?
 
+It means to think about apps as you build them.
```

Comment:
Thinking in react more so means thinking about how to break down your apps into components as modularly as possible but I'll take it